contracts/ExitHandler.sol

```diff
@@ -65,6 +105,238 @@ contract ExitHandler is DepositHandler {
     exitDuration = _exitDuration;
   }
 
+  function startLimboExit(bytes memory inTxData) public payable returns (bytes32 utxoId) { 
+
+    // exitor assumes tx to be canonical
+    require(msg.value >= exitStake, "Not enough ether sent to pay for exit stake");
+    TxLib.Tx memory transferTx = TxLib.parseTx(inTxData);
+    uint8 _outputIndex = 0;
+    
+    mapping(uint8 => LimboIn) public inputs;
+    mapping(uint8 => LimboOut) public outputs;
+
+    for (uint256 i =0; i < transferTx.outs.length; i++){
+      TxLib.Output memory out = transferTx.outs[i];
+      outputs[i] = LimboOut({
+        amount: out.value, 
+        owner: out.owner, 
+        isPegged: false, 
+        color: out.color, 
+        exitable: true });
+    }
+    for (uint256 i =0; i < transferTx.ins.length; i++){
+      TxLib.Input memory ins = transferTx.ins[i];
+      inputs[i] = LimboIn({
+        owner: out.owner, 
+        isPegged: false, 
+        exitable: true });
+    }    
+    
+    bytes32 inTxHash = keccak256(inTxData);
+
+    // for now assume outputIndex 0
```

Comment:
what should this be later? youngest input?

Reply:
If a tx has multiple inputs and multiple outputs then there will be multiple utxo's so right now i am handling one but a tx might have more so i how should i handle that scenario?
Also if a tx has multiple utxo's then should i write a combine function which will get me combined priority for the tx (i can use  `getERC20ExitPriority` to get individual utxo priority)

Reply:
Can i use youngest input in any way for this?